tendrl/commons/objects/definition/master.yaml

```diff
@@ -759,6 +759,63 @@ namespace.tendrl:
       value: alerting/clusters/$Alert.tags['integration_id']/$Alert.alert_id
       list: alerting/clusters/$Alert.tags['integration_id']
       help: "Cluster alerts"
+    UtilizationOnlyAlert:
```

Comment:
What is the connection with "Utilization" in the name? @shtripat @GowthamShanmugam

Reply:
@shtripat  i am also felt this name will confuse.. need to change this object name

Reply:
Ah my bad I meant `NotificationOnlyAlert` and not `UtilizationOnlyAlert` :(
This alert class represents alerts for which no clearing events would come later.